app/DBConnection.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const apiLocation = "../api/";
+const api = require(apiLocation + "database.js");
```

Comment:
Why store the `apiLocation` in a separate variable? It isn't reused and probably won't change in the future.